docs/RFCS/20220810_tenant_capabilities.md

```diff
@@ -0,0 +1,466 @@
+| `READ_SYSTEM_TIMESERIES`                  | Read timeseries for the KV-level metrics. [^ts]                                                                                                                                        |
+| `WRITE_TENANT_TIMESERIES`                 | Write SQL-level timeseries data. [^ts]                                                                                                                                                 |
+| `RANGE_LIST(RangeSpec...)`                | Inspect range descriptors for the specified ranges (can include "meta", "liveness" etc., see above; as well as "other tenants". A tenant's own ranges are always implicitly included.) |
+| `RANGE_MOVE(RangeSpec...)`                | Relocate or scatter replicas or leaseholders for the specified ranges to specific nodes (see above for range spec. A tenant's own ranges are not implicitly included.)                 |
+| `RANGE_SPLIT(RangeSpec...)`               | Manual split of specified ranges (see above for range spec. A tenant's own ranges are not implicitly included.)                                                                        |
+| `SET_SYSTEM_CLUSTER_SETTING(Settings...)` | Set the specified SystemOnly cluster settings. The capability may include a range of allowed values for each setting.                                                                  |
+| `GET_SYSTEM_CLUSTER_SETTING(Settings...)` | Get the specified SystemOnly cluster settings.                                                                                                                                         |
+| `GET_SYSTEM_ZONE_CONFIG(RangeSpec...)`    | Get the zone configs for non-tenant keyspaces (range type can be "meta", "timeseries", "liveness", etc. Refers to `SHOW ZONE CONFIGURATION FOR RANGE ...`).                            |
+| `SET_SYSTEM_ZONE_CONFIG(RangeSpec...)`    | Set the zone configs for non-tenant keyspaces (see above, refers to `ALTER RANGE ... SET ZONE CONFIGURATION`).                                                                         |
+| `CHANGE_SYSTEM_TENANT`                    | Set the `SystemTenantID` config key, to configure which tenant is authoritative for `settings`, `tenants` etc. [^st]                                                                   |
+
+[^ts]: See the separate proposal on [Timeseries access for tenants](https://github.com/cockroachdb/cockroach/pull/86524).
+
+[^st]: See section below [Deciding which tenant is responsible for hosting cluster configuration](#deciding-which-tenant-is-responsible-for-hosting-cluster-configuration).
+
+We will likely also need new capabilities to issue backups for another
+tenant than the tenant itself (such that a "system tenant" can run
+BACKUP FOR TENANT on a keyspace other than its own). Once the
+capability system is in place, we can have different teams consider
+the capabilities they need separately from each other.
+
+## Mechanisms
```

Comment:
One detail I don't see is how tenant processes observe their own capabilities. It's assumed below but there are no specifics.